example-app/src/main/cpp/backtrace-android.h

```diff
@@ -0,0 +1,332 @@
+#ifndef BACKTRACE_ANDROID_BACKTRACE_ANDROID_H
+#define BACKTRACE_ANDROID_BACKTRACE_ANDROID_H
+
+#include <jni.h>
+#include <string>
+#include <unordered_map>
+#include <android/log.h>
+// This is a header including additional Backtrace functionality
+// for NDK applications using the `backtrace-android` crash-reporting
+// library
+
+namespace Backtrace {
```

Comment:
@KrzaQ here I have namespaced everything in our public header. I believe there should not be any new globals exposed anywhere else. 

We don't have to worry about `native-lib.cpp`, that's just the code for the example app.